exercises/practice/phone-number/PhoneNumber.pm

```diff
@@ -7,6 +7,18 @@ use feature qw<say>;
 use Exporter qw<import>;
 our @EXPORT_OK = qw<clean_number>;
 
+my @errors = (
+    '11 digits must start with 1',
+    'must not be greater than 11 digits',
+    'must not be fewer than 10 digits',
+    'letters not permitted',
+    'punctuations not permitted',
```

Comment:
Could this be stated as "only digits permitted"?  This may be coming from upstream though, so may be irrelevant here.